src/lib/components/support.svelte

```diff
@@ -1,123 +1,162 @@
     import { organization } from '$lib/stores/organization';
     import { BillingPlan } from '$lib/constants';
     import { trackEvent } from '$lib/actions/analytics';
-    import { localeTimezoneName, utcHourToLocaleHour } from '$lib/helpers/date';
+    import { localeShortTimezoneName, utcHourToLocaleHour } from '$lib/helpers/date';
     import { upgradeURL } from '$lib/stores/billing';
+    import { Card } from '$lib/components/index';
+    import { app } from '$lib/stores/app';
 
     export let show = false;
 
     $: isPaid =
         $organization?.billingPlan === BillingPlan.PRO ||
         $organization?.billingPlan === BillingPlan.SCALE;
 
-    $: supportTimings = `${utcHourToLocaleHour('16:00')} - ${utcHourToLocaleHour('00:00')} ${localeTimezoneName()}`;
+    $: supportTimings = `${utcHourToLocaleHour('16:00')} - ${utcHourToLocaleHour('00:00')} ${localeShortTimezoneName()}`;
+
+    type SupportOption = {
+        cta?: string;
+        icon: string;
+        label: string;
+        link?: string;
+        description: string;
+        showSupport: boolean;
+    };
+
+    const supportOptions: SupportOption[] = [
+        {
+            showSupport: true,
+            icon: 'support',
+            label: 'Premium support',
+            description: 'Get priority email support from the Appwrite team'
+        },
+
+        {
+            icon: 'discord',
+            cta: 'Discord',
+            showSupport: false,
+            label: 'Community support',
+            link: 'https://appwrite.io/discord',
+            description: 'Get support from our community through Discord'
+        },
+        {
+            icon: 'github',
+            cta: 'Open issue',
+            showSupport: false,
+            label: 'Open GitHub issue',
+            link: 'https://github.com/appwrite',
+            description: 'Report a bug or pitch a new feature'
+        }
+    ];
 </script>
 
-{#if isCloud}
-    <section class="drop-section u-grid u-gap-24 u-padding-24">
-        <div>
-            <h4 class="eyebrow-heading-3">Premium support</h4>
-            {#if isPaid}
+<section class="drop-section support-section">
+    <h4 class="heading-level-6">Support</h4>
+
+    {#each supportOptions as option}
+        <Card
+            isTile
+            class="support-option-card u-flex u-flex-vertical u-gap-12"
+            style="border-radius: var(--border-radius-small, 8px); padding: 0.65rem;">
+            <div class="u-flex u-flex-vertical u-gap-4">
+                <h4 class="body-text-2 u-bold">{option.label}</h4>
+
                 <p class="u-line-height-1-5 u-margin-block-start-8">
-                    Get personalized support from the Appwrite team from <b>{supportTimings}</b>
+                    {option.description}
                 </p>
+            </div>
+
+            {#if option.showSupport}
+                <div class="u-flex u-gap-12 u-cross-center">
+                    {#if !isPaid}
```

Comment:
instead of looking at the plan for premium support, let's look at the current org's plan and check if `premiumSupport` is true.